### ISSUE TITLE
Evaluating constant expression yields wrong result for this expression

### ISSUE THREAD

**Issue by arnaudbos** [alygin/vscode-tlaplus#176]
Using the "Evaluate expression" or "Evaluate selected expression" I've tried this expression for some reason, and it did yield a wrong representation of the result in the Output window.

```tla
>> [1..5 -> {[a |-> {"a"}]}]
{<<[a |-> {"a"}], [a |-> {"a"}], [a |-> {"a"}]>>}
```

instead of 
```tla
{<<[a |-> {"a"}], [a |-> {"a"}], [a |-> {"a"}], [a |-> {"a"}], [a |-> {"a"}]>>}
```

I said that only the *representation* is wrong because the domain is correct
```tla
>> DOMAIN CHOOSE x \in [1..5 -> {[a |-> {"a"}]}] : TRUE
{1, 2, 3, 4, 5}
```

It may be an evaluation error for this specific expression, or some representation error maybe.

The output is actually similar for any number of elements in the inner function. But if the value is not a set then the 5 elements of the resulting set are correctly rendered:
```tla
>> [1..5 -> {[a |-> "a"]}]
{<<[a |-> "a"], [a |-> "a"], [a |-> "a"], [a |-> "a"], [a |-> "a"]>>}
```

This error does not occur in the TLA+ Toolbox.

At first glance I did not find a quick way to test this using the tla2tools.jar (the embedded one doesn't seem to be the latest released version) because I have no idea how to use it.  
If you'd like more details or if you want me to give this bugfix a try I'd be happy to help, but I could use some help with setting up a command-line repro-case (a documentation link maybe?) and/or pointers to thing to look at.

And thanks for the great plugin by the way.

**Comment by lemmy** [comment]
Fixed in https://github.com/alygin/vscode-tlaplus/pull/177

**Comment by alygin** [comment]
Fix available in the [1.5.2-alpha.1 prerelease](https://github.com/alygin/vscode-tlaplus/releases/tag/v1.5.2-alpha.1)

**Comment by alygin** [comment]
Fix published in v1.5.2
